src/get-started/java-binding.md

```diff
@@ -738,21 +738,21 @@ on possible configuration options.
 
 ### Java Logging
 
-Java logs are produced by Java Binding internal code and can also be
-used to monitor Java services internal logic.
+Logs produced by Java code (the framework and its dependencies,
+and the deployed services) are handled by Log4J framework.
+The services can use either [`Log4J`][log4j-home] or
+[`SLF4J`][slf4j-home] logging APIs.
 
 Java logging configuration is controlled by the configuration file
 specified by the `ejb-log-config-path` parameter. If no file was
 provided, the logs are disabled. Exonum Java package provides an
 example `log4j-fallback.xml` configuration file that can be found
-at the installation directory.
+at the installation directory. With this file `INFO`-level messages
+are output to stdout.
```

Comment:
Past participle appears to be 'ouputted': https://www.oxfordlearnersdictionaries.com/definition/english/output_2

Maybe printed is better?